.github/workflows/ci.yml

```diff
@@ -31,28 +31,26 @@ env:
   SWIFTLINT_VERSION: 0.57.0
   SWIFTFORMAT_VERSION: 0.54.6
 jobs:
-  swift-tests:
-    timeout-minutes: 15
-    runs-on: ubuntu-latest
-    strategy:
-      fail-fast: false
-      matrix:
-        swift_version: ['6.0']
-        os_version: ['jammy']
-    container:
-      image: swift:${{ matrix.swift_version }}-${{ matrix.os_version }}
-    name: swift ${{ matrix.swift_version }} tests
-    steps:
-    - name: Swift version
-      run: swift --version
-    - name: Checkout repository
-      uses: actions/checkout@v4
-    - name: Install jemalloc
-      run: apt-get update && apt-get install -y libjemalloc-dev
-    - name: Run tests
-      run: swift test --configuration release --parallel
-    - name: Run snippets
-      run: >
+  soundness:
+    name: soundness
+    uses: fboemer/github-workflows/.github/workflows/soundness.yml@fboemer/append-docc-plugin-only-when-needed
+    with:
+      # https://github.com/swiftlang/swift-package-manager/issues/8103
+      api_breakage_check_enabled: false
+      format_check_enabled: false
+      license_header_check_enabled: false
+      python_lint_check_enabled: false
+      yamllint_check_enabled: false
+  tests:
+    name: swifttests
+    uses: swiftlang/github-workflows/.github/workflows/swift_package_test.yml@main
+    with:
+      enable_windows_checks: false
```

Comment:
Ran into some issues with Benchmark:
https://github.com/apple/swift-homomorphic-encryption/actions/runs/14010334789/job/39229328473?pr=187

Reply:
Even when disabling benchmarking, I ran into linking issues, seemingly due to having multiple `main.swift` files:
https://github.com/apple/swift-homomorphic-encryption/actions/runs/14059356315/job/39366144668#step:9:1144